sources/install/package_wordlists.sh

```diff
@@ -106,6 +105,38 @@ function install_genusernames() {
     add-to-list "genusernames,https://gitlab.com/-/snippets/2480505/raw/main/bash,GenUsername is a Python tool for generating a list of usernames based on a name or email address."
 }
 
+
+
+function install_onelistforall() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing onelistforall"
+    wget https://raw.githubusercontent.com/six2dez/OneListForAll/main/onelistforallmicro.txt -P /opt/lists/
+    wget https://raw.githubusercontent.com/six2dez/OneListForAll/main/onelistforallshort.txt -P /opt/lists/
+    add-test-command "[[ -f '/opt/lists/onelistforallshort.txt' ]]"
+    add-to-list "onelistforall,https://github.com/six2dez/OneListForAll,Rockyou for web fuzzing"
+}
+
+function install_rules(){
+    colorecho "Add rules"
+    wget https://github.com/NSAKEY/nsa-rules/blob/master/_NSAKEY.v2.dive.rule -P /opt/rules/
+    wget https://github.com/praetorian-inc/Hob0Rules/blob/master/d3adhob0.rule -P /opt/rules/
+    wget https://github.com/praetorian-inc/Hob0Rules/blob/master/hob064.rule -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/hashesorg.v6/pantagrule.hashorg.v6.hybrid.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/hashesorg.v6/pantagrule.hashorg.v6.one.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/hashesorg.v6/pantagrule.hashorg.v6.popular.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/hashesorg.v6/pantagrule.hashorg.v6.random.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/hashesorg.v6/pantagrule.hashorg.v6.raw1m.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.hashorg.royce/pantagrule.hybrid.royce.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.hashorg.royce/pantagrule.one.royce.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.hashorg.royce/pantagrule.popular.royce.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.hashorg.royce/pantagrule.random.royce.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.v5/pantagrule.private.v5.one.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.v5/pantagrule.private.v5.popular.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.v5/pantagrule.private.v5.random.rule.gz -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/tree/master/rules/private.v5/pantagrule.private.v5.hybrid.rule.gz -P /opt/rules/
```

Comment:
All files are HTML documents, so there's a problem with your download. You need to download raw files :

`https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/hashesorg.v6/pantagrule.hashorg.v6.hybrid.rule.gz`

```bash
> head -n 20 pantagrule.hashorg.v6.hybrid.rule.gz


<!DOCTYPE html>
<html
  lang="en"

  data-color-mode="auto" data-light-theme="light" data-dark-theme="dark"
  data-a11y-animated-images="system" data-a11y-link-underlines="true"

  >



  <head>
    <meta charset="utf-8">
  <link rel="dns-prefetch" href="https://github.githubassets.com">
```